### ISSUE TITLE
doom_cute_icon does not work

### ISSUE THREAD

**Issue by pythonian23** [Mstrodl/elcord#88]
Setting `elcord-editor-icon` to `doom_cute_icon` doesn't work and the image isn't shown, but it works for other images.
```elisp
(setq elcord-editor-icon "doom_cute_icon")
(elcord-mode)
```
![image](https://github.com/Mstrodl/elcord/assets/63051526/2854f965-c503-4619-946f-752a11053700)
```elisp
(setq elcord-editor-icon "doom_cute_icon")
(elcord-mode)
```
![image](https://github.com/Mstrodl/elcord/assets/63051526/c7c2be87-b393-48ae-842e-73929eb00848)

**Comment by pythonian23** [comment]
I think it may have something to do with the file or image size, but that's just my uneducated guess

**Comment by Zulu-Inuoe** [comment]
Hum. I just checked the portal and it looks like the icon wasn't uploaded. Going to check the auto-upload script for anything

**Comment by Zulu-Inuoe** [comment]
Interesting. I hadn't noticed that d0c9ace493d088bc70f7422705ff27dfcf162cca made it so that icons are fetched directly from GitHub. I didn't know about that as an option.
I'll need to dig into this further. Your suggestion of the file size might be right

**Comment by stalecu** [comment]
Any progress on this? This issue is still here.

**Comment by thetonk** [comment]
Another Doom Emacs user facing the same issue here. After some quick investigation, there is another editor icon that is not loading, it is not only the doom_cute icon. During my tests, I noticed the following editor icons WORK as expected;

- emacs_pen_icon
- spacemacs_icon
- emacs_material_icon
- emacs_icon
- doom_icon
- emacs_legacy_icon

The following icons DO NOT work;
- doom_cute_icon
- emacs_dragon_icon

The common ground between those problematic two is that both have 1024x1024 dimensions. However, spacemacs_icon, emacs_legacy_icon, emacs_pen_icon have 1024x1024 dimensions as well.... Perhaps resizing all icons to 512x512 which is the required minimum size for Discord Rich Presence images could help?

**Comment by Mstrodl** [comment]
Figured it out. Looks like I wasn't thorough enough when converting the icons to use an external link, so it was still using only the assets.

**Comment by Mstrodl** [comment]
Should be fixed in e2775f40ec55dfdceea83d535dff77d60534b6bc, feel free to reopen if not! :)
